libcrux-ml-dsa/benches/bench_utils.rs

```diff
@@ -0,0 +1,170 @@
+use rand::RngCore;
+
+#[allow(unused)]
+pub(crate) fn random_array<const L: usize>() -> [u8; L] {
+    let mut rng = rand::rngs::OsRng;
+    let mut seed = [0; L];
+    rng.try_fill_bytes(&mut seed).unwrap();
+    seed
+}
+
+#[allow(unused)]
+pub(crate) fn print_time(label: &str, d: std::time::Duration) {
+    let micros = d.as_micros();
+    let time = if micros < MILLI_PER_ITERATION_THRESHOLD {
+        format!("{} μs", micros / ITERATIONS as u128)
+    } else if micros < SECOND_PER_ITERATION_THRESHOLD {
+        format!(
+            "{:.2} ms",
+            (micros as f64 / (MICROS_PER_MILLI * ITERATIONS as f64))
+        )
+    } else {
+        format!(
+            "{:.2}s",
+            (micros as f64 / (MICROS_PER_SECOND * ITERATIONS as f64))
+        )
+    };
+    let space = if label.len() < 6 {
+        "\t\t".to_string()
+    } else {
+        "\t".to_string()
+    };
+
+    println!("{label}:{space}{time}");
+}
+
+pub(crate) const ITERATIONS: usize = 100_000;
+#[allow(unused)]
```

Comment:
Is this `allow` needed? This should be used?

Reply:
If I remove this I get `dead_code` warning. It's weird, right? Since I don't get the same for `ITERATIONS`.

Reply:
weird, then leave it in